src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -359,16 +359,19 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         pop["age_bin"] = pd.cut(pop["child_age"], self.age_bins["age_start"])
         pop = pop.sort_values(["sex_of_child", "child_age"])
 
+        num_sexes = pop.sex_of_child.nunique()
+        assert num_sexes == 2
```

Comment:
I really dislike assertions in production code (although one could argue this is really artifact generation code and I would agree with them). Maybe raise a ValueError instead.

Reply:
We talked a bit about this in stand up today. It seems like the only thing we really lose with an assert here is the type error as long as we add an informative message explaining the error.